pages/docs/contributing/maintainers.mdx

```diff
@@ -0,0 +1,87 @@
+import DocsLayout from "../../../components/layouts/DocsLayout.js";
+
+## Maintainers
+
+- Only assign issues with the label `status: ready for dev`
+- Each person should only be assigned 1 open issue at a time
+
+---
+
+> `avatar` json property is included
+
+This is no longer used, please leave an inline comment with the code sugestion to remove it - this will allow another maintainer to accept the changes and we can get the Pull Request merged without delay.
```

Comment:
```suggestion
**Note:** This is no longer used, please leave an inline comment with the code sugestion to remove it - this will allow another maintainer to accept the changes and we can get the Pull Request merged without delay.
```